test_tools/sigma_verify_rules.py

```diff
@@ -76,8 +116,27 @@ def run_verifier(rules_path, config_file_path):
                     continue
 
                 rule_file_path = os.path.join(dirpath, rule_filename)
-                parsed_rule = sigma_util.get_sigma_rule(
-                    rule_file_path, sigma_config)
+
+                block_because_csv = False
+                for item in ignore['path']:
```

Comment:
ignore is a dataframe, so I would just do

```python
ignore_list = list(ignore['path'].unique())
```

and then you can iterate over that, so you'll only need to do that once, not once for every filepath and for every rule

Reply:
ok, so what you are doing is iterating over all the ignore paths and checking each one of the ignore paths if they are in the rule file path... is not possible to do something like `if rule_file_path in ignore_list`? or normalize the rule path first and check if it is there?

Reply:
yes, much better